src/components/FilterDropdown.jsx

```diff
@@ -26,9 +26,11 @@ function FilterDropdown(props) {
     <FormControl variant="outlined" className={classes.formControl}>
       <InputLabel>{label}</InputLabel>
       <Select value={choice} onChange={selectChoice} label={label}>
-        <MenuItem value="">
-          <em>None</em>
-        </MenuItem>
+        {removeNone ? null : (
```

Comment:
I think a better way would be `{removeNone || ...}`.

The way `||` works is that it first evaluates the first thing before evaluating the other. So you want it to evaluate `removeNone` as `true` or create the `None` option otherwise.